### DISCUSSION TITLE
Is this possible to Change This Type Of App Package Name In Sketchware Pro

### DISCUSSION THREAD

**Discussion by rajendrakrdangi** [Sketchware-Pro/Sketchware-Pro#1054] [General]
I have an app with the package name 'com.new.game' on the Play Store, but when attempting to change the app's package name in Sketchware Pro, I encounter an error stating that  can't use reserved keywords.
![Screenshot_20231231-001956](https://github.com/Sketchware-Pro/Sketchware-Pro/assets/59394255/24cd6330-26f5-4682-a4b3-0ed3e05a813b)
![Screenshot_20231231-002531](https://github.com/Sketchware-Pro/Sketchware-Pro/assets/59394255/702ab16f-d3a1-4fe3-9fc4-73f757b91d22)

**Comment by j-px** [comment]
Apparently package names can't contain reserved keywords like `new` (try it for yourself in any Java compiler!), so the limitation is there for a reason.

**Reply by rajendrakrdangi** [reply]
Okay Thank You. but how to do myself can you guide me please?

**Reply by j-px** [reply]
do what

**Reply by rajendrakrdangi** [reply]
nothing, i got your point
